plugins/woocommerce/client/blocks/tests/e2e/utils/editor/editor-utils.page.ts

```diff
@@ -149,7 +155,34 @@ export class Editor extends CoreEditor {
 			.waitFor();
 	}
 
-	async revertTemplate( { templateName }: { templateName: string } ) {
+	// Since WP 6.9 templates (and only templates) are handled differently due to template activation
+	// and require different flow to be reverted.
+	async revertTemplateSiceWP69( { templateName }: { templateName: string } ) {
```

Comment:
There is a typo here.


```suggestion
	async revertTemplateSinceWP69( { templateName }: { templateName: string } ) {
```

Reply:
Can't believe claude had it wrong!! 😂